comms.py

```diff
@@ -0,0 +1,62 @@
+from mcu import *
+
+class Communications:
+    
+    PWM_MIN = 1000
+    PWM_MID = 1500
+    PWM_MAX = 2000
+    
+    MOTOR_CLAW = 0 
+    MOTOR_FRONT = 1
+    MOTOR_RIGHT = 2
+    MOTOR_LEFT = 3 
+    MOTOR_BACK = 4
+    
+    INITIAL_PERCENT = 100
+    MULTIPLIER_PERCENT = 2
```

Comment:
Multiplier percentage should be fed in as a parameter when initializing the class object.